lib/Controller/InternalAddressController.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Mail\Controller;
+
+use OCA\Mail\AppInfo\Application;
+use OCA\Mail\Http\JsonResponse;
+use OCA\Mail\Http\TrapError;
+use OCA\Mail\Service\InternalAddressService;
+use OCP\AppFramework\Controller;
+use OCP\AppFramework\Http;
+use OCP\IRequest;
+
+class InternalAddressController extends Controller {
+	private ?string $uid;
+
+	public function __construct(IRequest        $request,
+		?string                                 $UserId,
```

Comment:
```suggestion
		?string                                 $userId,
```

https://docs.nextcloud.com/server/latest/developer_manual/basics/dependency_injection.html#predefined-core-services